Model/Service/SendEmail.php

```diff
@@ -52,30 +52,36 @@ public function __construct(
      */
     public function with($reportHtml)
     {
-        $this->inlineTranslation->suspend();
-        $sender = [
-            'name'  => $this->escaper->escapeHtml($this->getMailSenderClearName()),
-            'email' => $this->escaper->escapeHtml($this->getMailSenderAddress()),
-        ];
-        $transportBuilder = $this->transportBuilder->setTemplateIdentifier('security_csp_report')->setTemplateOptions(
-            [
-                'area'  => \Magento\Framework\App\Area::AREA_ADMINHTML,
-                'store' => \Magento\Store\Model\Store::DEFAULT_STORE_ID,
-            ]
-        )->setTemplateVars(
-            [
-                'body' => $reportHtml,
-            ]
-        )->setFrom($sender);
+        $recepients = [];
+        try {
+            $this->inlineTranslation->suspend();
+            $sender = [
+                'name'  => $this->escaper->escapeHtml($this->getMailSenderClearName()),
+                'email' => $this->escaper->escapeHtml($this->getMailSenderAddress()),
+            ];
+            $transportBuilder =
+                $this->transportBuilder->setTemplateIdentifier('security_csp_report')->setTemplateOptions(
+                    [
+                        'area'  => \Magento\Framework\App\Area::AREA_ADMINHTML,
+                        'store' => \Magento\Store\Model\Store::DEFAULT_STORE_ID,
+                    ]
+                )->setTemplateVars(
+                    [
+                        'body' => $reportHtml,
+                    ]
+                )->setFrom($sender);
 
-        $recepients = $this->getMailRecipients();
-        foreach ($recepients as $recipient) {
-            $transportBuilder->addTo($recipient);
+            $recepients = $this->getMailRecipients();
+            foreach ($recepients as $recipient) {
+                $transportBuilder->addTo($recipient);
+            }
+            $transportBuilder->getTransport()->sendMessage();
+        } catch (EmailSendException $exception) {
```

Comment:
If you don't do anything with the exception but re-throwing, you can omit  the catch block, `try {} finally {}` also works without `catch`

Reply:
big aha here